unix/Makefile

```diff
@@ -74,10 +77,14 @@ ifeq ($(MICROPY_FORCE_32BIT),1)
 # starting with linux-libc-dev:i386
 ifeq ($(MICROPY_PY_FFI),1)
 ifeq ($(UNAME_S),Linux)
+ifeq ($(UNAME_M), armv7l)
+CFLAGS_MOD += -I/usr/include/arm-linux-gnueabihf
```

Comment:
I don't. This is just in case you need to set the bit and are building on an ARM platform.